app/script/calling/CallMessageBuilder.js

```diff
@@ -23,137 +23,137 @@ window.z = window.z || {};
 window.z.calling = z.calling || {};
 
 z.calling.CallMessageBuilder = (function() {
-  const _build_call_message = function(type, response, session_id, additional_payload) {
-    const call_message_et = new z.calling.entities.CallMessage(type, response, session_id);
+  const buildCallMessage = function(type, response, sessionId, additionalPayload) {
```

Comment:
Private functions remain to be tagged as private indicated by the underscore. Please do not remove that.

Reply:
`_buildCallMessage`, is it right way?

Reply:
Yes